File: CTDK_Python/073/_073.py
x = float(input("Nhap x: "))
n = int(input("Nhap n: "))

s = 0
t = 1
m = 0
i = 1
while(i <= n):
    t = t * x
    m = m + i
    s = s + t / m
    i = i + 1

print("Tong la: ", s)

File: CTDK/073/073.cpp
#include <iostream>
using namespace std;

int main()
{
	float x;
	cout << "Nhap x: ";
	cin >> x;

	int n;
	cout << "Nhap n: ";
	cin >> n;

	float s = 0;
	float t = 1;
	int m = 0;
	int i = 1;
	while (i <= n)
	{
		t = t * x;
		m = m + i;
		s = s + t/m;
		i = i + 1;
	}
	cout << "Tong la: " << s << endl;
	return 0;
}